packages/data/src/world/mm/overworld.yml

```diff
@@ -877,7 +877,7 @@
   exits:
     "Termina Field": "true"
   locations:
-    "Termina Field Bio Baba Grotto": "has_mask_zora"
+    "Termina Field Bio Baba Grotto": "underwater_walking"
```

Comment:
Gonna need a way to hit the hive down. so either hook or bow, maybe bombs too if you have the bio baba soul.

Reply:
`has_mask_zora || (underwater_walking && has_weapon_range)` good?